config/acme/machines/config_machines.xml

```diff
@@ -1399,6 +1400,7 @@
     <SUPPORTED_BY>acme</SUPPORTED_BY>
     <GMAKE_J>8</GMAKE_J>
     <MAX_TASKS_PER_NODE>16</MAX_TASKS_PER_NODE>
+    <NUM_SPARE_NODES>2</NUM_SPARE_NODES>
```

Comment:
This should probably be a percentage of the total nodes instead of a fixed value.